pandas/util/_test_decorators.py

```diff
@@ -115,6 +117,10 @@ def skip_if_no(package: str, min_version: str | None = None) -> pytest.MarkDecor
     locale.getlocale()[0] != "en_US",
     reason=f"Set local {locale.getlocale()[0]} is not en_US",
 )
+skip_if_wasm = pytest.mark.skipif(
+    (sys.platform == "emscripten") or (platform.machine() in ["wasm32", "wasm64"]),
```

Comment:
Could you use your `WASM` variable here?

Reply:
Addressed in [`cc233e4`](https://github.com/pandas-dev/pandas/pull/57896/commits/cc233e4c88725440528c1b1aa690843c1d756508), along with a few other style changes across files (I activated the linter locally).